extension/src/cdp/window.ts

```diff
@@ -0,0 +1,35 @@
+import { BrowserExtensionClient } from '../messaging'
+
+let wasFocused = false
+
+export function trackTabFocus(client: BrowserExtensionClient) {
```

Comment:
CDP doesn't have a way of tracking which window or tab is currently active, so we have to report this from the browser itself.